pkg/controller/plan/kubevirt.go

```diff
@@ -661,7 +672,9 @@ func (r *KubeVirt) createPodToBindPVCs(vm *plan.VMStatus, pvcNames []string) err
 			RestartPolicy: core.RestartPolicyNever,
 			Containers: []core.Container{
 				{
-					Name:    "main",
+					Name: "main",
+					// For v2v the consumer pod is only when we are using cold migration on el9.
+					// In that case, it might be useful to cache the image now and save time for the conversion pod.
```

Comment:
1. how about separating this to another PR that is correlated with the issue about this?
2. please add the missing part to the first sentence ("used"?)
3.  how about
```suggestion
					// In that case, we could benefit from pulling the image of the conversion pod so it will be present on the node.
```

Reply:
done - PR #245